editor/filesystem_dock.cpp

```diff
@@ -94,15 +94,17 @@ bool FileSystemDock::_create_tree(TreeItem *p_parent, EditorFileSystemDirectory
 	// Create all items for the files in the subdirectory.
 	if (display_mode == DISPLAY_MODE_TREE_ONLY) {
 		String main_scene = ProjectSettings::get_singleton()->get("application/run/main_scene");
+
+		// Build the list of the files to display
+		List<FileInfo> filelist;
```

Comment:
Is there a reason this variable is named `filelist` instead of `file_list`?

Reply:
No really, I kind of imitated another part of the code.
Anyway, I renamed everything to file_list. :)